source/1/1.1/retrieving_the_image.rst

```diff
@@ -1,4 +1,17 @@
 Chapter 1.1: Retrieving the image
 =================================
 
-Before you install Lubuntu, you need to retrieve the correct image.
+Before you install Lubuntu, you need to retrieve the correct image. Lubuntu offers two different kinds of images alternate and desktop images. Lubuntu is also for different architextures of cpus. Lubuntu releases a new version every six months with every two years having a long term support release. 
+
+Choosing between alternate and desktop 
+--------------------------------------
+Alternate images are mainly intended for older lower spec hardware and uses  a non graphical installer. The desktop version offers a live session to try out lubuntu and make sure all of your hardware works with Lubuntu before you install it on that computer. The installer on the desktop is a graphical installer where you can use mouse and keyboard and is more user friendly but takes more resources than to run Lubuntu itself. If your computer has less than 512 MB of RAM it is recommended to use the alternate installer as this will result in not having the installer run out of memory and fail to install. The alternate installer does not have have a live session however.  
+
+Choosing the correct architexture
+---------------------------------
+Lubuntu offers images for three different Architextures AMD 64, i386, and powerpc. PowerPc is for older macbook hardware before 2006 and IBM openpower systems. The remaining two version are i386 the 32 bit version and AMD 64. 32 bit will work if you are unusure. The AMD64 will work with newer in intel and amd processors. Often computers bought from a store will come with a branding sticker saying what cpu is used in the computer. If it lists core i3, core i5,  core i7 on this sticker or core 2 solo, core 2 duo, core 2 quad, or core 2 extreme then you have 64 bit intel processor.   If your pc says pentium m, pentium III, pentium IV, then you will need the 32 bit versions. Earlier Intel atom based processors  were 32 bit on netbooks but some of the later ones are 64 bit. If you  have an AMD processor if it is says an fx processor,  or athlon 64 processor it is definetly 64 bit. Some athlon and sempron branded amd hardware are 32 or 64 bit but AMD computers had 64 bit earlier on so it will need an old AMD cpu to need 32 bit. However 32 bit can be run on 64 bit hardware One reason you may want to do this is to use less ram than you would normally use. IF you have more than 2 GB of ram this is not normally a problem and is not really a consideration. 
```

Comment:
Do it like this:
```
Lubuntu offers images for three (3) different architectures:
 - amd64
 - i386
 - PowerPC
```
You could then do subheaders to talk about each architecture.